container/build.sh

```
@@ -114,7 +114,7 @@ SGLANG_BASE_IMAGE_TAG="25.01-cuda12.8-devel-ubuntu24.04"
 VLLM_V1_BASE_IMAGE="nvcr.io/nvidia/cuda-dl-base"
 VLLM_V1_BASE_IMAGE_TAG="25.01-cuda12.8-devel-ubuntu24.04"
```

Shellcheck (0.10.0)

[warning] VLLM_V1_BASE_IMAGE_TAG appears unused. Verify use (or export if used externally).

(SC2034)